extension/src/plots/model/collect.ts

```diff
@@ -508,15 +508,16 @@ const updateDatapoints = (
   fields: string[]
 ): unknown[] =>
   selectedRevisions
-    .flatMap(revision =>
-      revisionData?.[revision]?.[path].map(data => {
+    .flatMap(revision => {
+      const datapoints = revisionData?.[revision]?.[path] || []
+      datapoints.map(data => {
```

Comment:
Test is failing because this needs to be returned...

Reply:
Fixes the issue:

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/37993418/202403574-fa3e490e-861c-48cb-b0bd-bff5dc39891b.png">

Reply:
![image](https://user-images.githubusercontent.com/37993418/202409802-66a14614-d2d5-42cc-a9ce-290b4bd336ff.png)